examples/protocol_specification_ex/ledger_api.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: ledger_api
+author: fetchai
+version: 0.1.0
+description: A protocol for ledger APIs requests and responses.
+license: Apache-2.0
+aea_version: '>=0.4.0, <0.5.0'
+speech_acts:
+  get_balance:
+    ledger_id: pt:str
+    address: pt:str
+  transfer:
+    ledger_id: pt:str
+    destination_address: pt:str
+    amount: pt:int
+    tx_fee: pt:int
+    tx_nonce: pt:int
+    data: pt:dict[pt:str, pt:str]
+
+  is_transaction_settled:
+    ledger_id: pt:str
+    tx_digest: pt:str
+  is_transaction_valid:
+    ledger_id: pt:str
+    tx_digest: pt:str
+
+  get_transaction_receipt:
+    ledger_id: pt:str
+    tx_digest: pt:str
+
+  generate_tx_nonce:
+    ledger_id: pt:str
```

Comment:
I think we can move this out. This was something @Totoual added as a hack at some point. It should be a separate helper functionality rather than part of the API.